unified-codes/tools/scripts/dgraph/migration/v2/import.js

```diff
@@ -0,0 +1,294 @@
+
+  const categoryCode = row.getCell(12).value;
+  const productCode = row.getCell(13).value;
+  const routeCode = row.getCell(14).value;
+  const doseFormCode = row.getCell(15).value;
+  const doseQualificationCode = row.getCell(16).value;
+  const doseStrengthCode = row.getCell(17).value;
+  const doseUnitCode = row.getCell(18).value;
+  const immediatePackagingCode = row.getCell(19).value;
+  const packSizeCode = row.getCell(20).value;
+  const outerPackagingCode = row.getCell(21).value;
+
+  // Properties
+  const gs1 = row.getCell(22).value;
+  const rxNavUC2 = row.getCell(23).value;
+  const rxNavUC67 = row.getCell(24).value;
+  const atcUC2 = row.getCell(25).value;
+  const dddUC2 = row.getCell(26).value;
+  const atcUC5 = row.getCell(27).value;
+  const dddUC5 = row.getCell(28).value;
+  const emlUC2 = row.getCell(29).value;
+  const emlUC67 = row.getCell(30).value;
+  const unspsc = row.getCell(31).value;
+  const nzulmUC2 = row.getCell(32).value;
+  const nzulmUC67 = row.getCell(33).value;
+  const snomed = row.getCell(34).value;
+  const drugbank = row.getCell(35).value;
+  const usfda = row.getCell(36).value;
+
+  const productDefinition = [
+    {
+      type: 'Product',
+      name: productName,
+      code: productCode,
+      properties: [
+        { type: 'code_nzulm', value: nzulmUC2 },
+        { type: 'code_rxnav', value: rxNavUC2 },
+        { type: 'code_unspsc', value: unspsc },
+        { type: 'code_who_atc', value: atcUC2 },
+        { type: 'ddd', value: dddUC2 },
+        { type: 'who_eml', value: emlUC2 },
+      ],
+    },
+    {
+      type: 'Route',
+      name: routeName,
+      code: routeCode,
+      properties: [],
+    },
+    {
+      type: 'DoseForm',
+      name: doseFormName,
+      code: doseFormCode,
+      properties: [],
+    },
+    {
+      type: 'DoseQualification',
+      name: doseQualificationName,
+      code: doseQualificationCode,
+      properties: [
+        { type: 'code_who_atc', value: atcUC5 },
+        { type: 'ddd', value: dddUC5 },
+      ],
+    },
+    {
+      type: 'DoseStrength',
+      name: doseStrengthName,
+      code: doseStrengthCode,
+      properties: [],
+    },
+    {
+      type: 'DoseUnit',
+      name: doseUnitName,
+      code: doseUnitCode,
+      properties: [],
+    },
+  ];
+
+  const [product] = productDefinition;
+
+  if (product.name.indexOf('/') > -1) {
+    combinationDrugs.push({
+      productName,
+      productCode,
+      combinations,
+    });
+  }
+
+  const synonymsArray = synonyms ? synonyms.split(',') : [];
+  await insertProduct(product, categoryCode, synonymsArray);
+
+  let parentIndex = 0;
+  let childIndex = 1;
+  while (childIndex < productDefinition.length) {
+    if (productDefinition[childIndex].name && productDefinition[childIndex].code) {
+      await insertChild(productDefinition[parentIndex], productDefinition[childIndex]);
+      parentIndex = childIndex;
+    }
+    childIndex++;
+  }
+};
```

Comment:
A little bit hard to follow... but works! Again, might benefit from being extracted out into a dedicated class method, just to make it a bit more modular.

As usual, not a priority atm!